### PR TITLE
Release operation definition

### PR DESCRIPTION
**Pull request by sliver007**

### CODE REVIEW

File: input/vocabulary/valueset-knowledgeArtifact-release-version-behavior.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  "url": "http://hl7.org/fhir/uv/cmi/ValueSet/knowledgeArtifact-release-version-behavior",
+  "version": "3.0.0",
+  "name": "KnowledgeArtifactReleaseVersionBehavior",
+  "title": "Knowledge Artifact Release Version Behavior Codes",
+  "status": "active",
+  "experimental": false,
+  "date": "2023-02-03T23:21:03-06:00",
+  "publisher": "Clinical Decision Support Workgroup",
+  "description": "Behaviors of 'version' parameter application in the $release operation.",
+  "jurisdiction": [
```
Comment (brynrhodes) [inline input/vocabulary/valueset-knowledgeArtifact-release-version-behavior.json]: Remove the jurisdiction here, this is a UV IG

File: input/vocabulary/codesystem-knowledgeArtifact-release-version-behavior.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  "url": "http://hl7.org/fhir/uv/cmi/CodeSystem/knowledgeArtifact-release-version-behavior",
+  "version": "3.0.0",
+  "name": "KnowledgeArtifactReleaseVersionBehavior",
+  "title": "Knowledge Artifact Release Version Behavior Codes",
+  "status": "active",
+  "experimental": false,
+  "date": "2023-02-03T23:21:03-06:00",
+  "publisher": "Clinical Decision Support Workgroup",
+  "description": "Behaviors of 'version' parameter application in the $release operation.",
+  "jurisdiction": [
```
Comment (brynrhodes) [inline input/vocabulary/codesystem-knowledgeArtifact-release-version-behavior.json]: Remove the jurisdiction here, this is a UV IG

File: input/resources/operationdefinition-knowledgeartifact-release.xml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        </telecom>
+        <telecom>
+            <system value="email"/>
+            <value value="fhir@lists.hl7.org"/>
+        </telecom>
+    </contact>
+    <description value="The release operation supports updating the status of an existing draft artifact to active. The operation sets the date element of the resource and pins versions of all direct and transitive references. Child artifacts (i.e. artifacts that compose the existing artifact) are also released, recursively."/>    
+    <code value="knowledge-artifact-release"/>
+    <comment value="The release operation supports the ability of a repository to transitively gather components and dependencies of the artifact and record those as dependencies on the artifact itself while updating the status of all components to 'active'. For both components and dependencies if versions are not specified in the relevant reference, the operation will infer the version to be used by either locating a current 'draft' version of the artifact being reference or locating the latest active version in the case of dependencies."/>
+    <resource value="Resource"/>
```
Comment (brynrhodes) [inline input/resources/operationdefinition-knowledgeartifact-release.xml:25]: Add that the version determined should be recorded in the "manifest"

**Review by brynrhodes — CHANGES_REQUESTED** [review]
Looks great other than some clarification on the release behavior and removing the US jurisdiction reerences.

File: input/resources/operationdefinition-knowledgeartifact-release.xml (hunk 3 above)
Comment (sliver007) [reply input/resources/operationdefinition-knowledgeartifact-release.xml:25]: Description and comment have both been updated significantly - beyond just the requested mention of recording references in the manifest. Please review the comment value closely.

**Pull request merged**